webapp/components/TopNav.tsx

```diff
@@ -0,0 +1,58 @@
+import styles from '../styles/TopNav.module.scss';
+import { FcSearch } from 'react-icons/fc';
+import { FiUser } from 'react-icons/fi';
+import Link from 'next/link';
+import { useState } from 'react';
+
+const TopNav = () => {
```

Comment:
We can rename `TopNav` to `Header`? Since we already have side navigation. Plus our `TopNav` doesn't solely do navigating. It does searching too.

Reply:
done